src/en/news/blog/2022/autoscaler_tuning/index.md

```diff
@@ -0,0 +1,143 @@
+   
+The `--bulk` flag can now be used to indicate that a pool will be large, and that it should start out with a large amount of PGs. Users can set the `--bulk` flag both when creating a new pool and on existing pools. Any pools created without the `--bulk` flag will keep the default `scale-up` behavior.
+
+To set the `--bulk` flag during pool creation, use `ceph osd pool create <pool-name> --bulk`. To set or unset the `--bulk` flag on an existing pool, use `ceph osd pool set <pool-name> bulk <true/false/1/0>`. To get the `--bulk` flag status on an existing pool, use `ceph osd pool get <pool-name> bulk`.
+
+### If you know the usage ratio of a pool ahead of time, use the pg\_num option before creation.
+
+When a cluster or pool is first created, only a small fraction of the cluster capacity is consumed. As a result, your system may underestimate the number of PGs a cluster or pool will need. If you know the usage ratio of your cluster or pool ahead of time, use the `pg_num` option before pool creation. This will help your cluster account for the appropriate amount of PGs in advance.
+
+To check the `pg_num` value for a pool, use `ceph osd pool autoscale-status` and look under the `PG_NUM` column. To specify `pg_num` when creating a pool, use `ceph osd pool create <pool_name> <pg_num>`. To set the minimum number of pgs allowed in a pool, use `ceph osd pool set <pool-name> pg_num_min <pg_num>`. To set the maximum number of pgs allowed in a pool, use `ceph osd pool set <pool-name> pg_num_max <pg_num>`.
+
+### Use the target\_size\_ratio setting if you expect a pool to consume more cluster capacity relative to other pools.
+ 
+The target ratio of a pool can be specified using the `target_size_ratio` setting. This setting is especially helpful if you know a pool should have a certain ratio relative to other pools. The target ratio of each pool should be 0 by default, unless it is specified otherwise. The more target ratio you give a pool, the larger the PGs you are expecting the pool to have.
+
+To check the target ratio for a pool, use `osd pool autoscale-status` and look under the `TARGET RATIO` column. The `target_size_ratio` can be set on an existing pool with `ceph osd pool set <pool_name> target_size_ratio <ratio>`. To set the `target_size_ratio` during pool creation, use `ceph osd pool create --target-size-ratio <ratio>`.
+
+### Use the noautoscale flag to globally toggle the autoscaler for all pools.
+
+The autoscaler can now be turned on and off globally for all pools with the `noautoscale` flag. By default, this flag is set to false, and the default `pg_autoscale` mode, which is `on`, remains the same for each pool.
+
+If you want to turn the autoscaler on for every pool, use `ceph osd pool set noautoscale`. If you want to turn off the autoscaler for every pool, use `ceph osd pool unset noautoscale`. To get the current value of the noautoscale flag, use `ceph osd pool get noautoscale`.
```

Comment:
the statement should be opposite to what it has now as the flag name is *no*autoscale :)

Reply:
whoops yep, you're right.